apache/ozone
# HDDS-8971. Example integration with Iceberg, Spark and Trino

## What changes were proposed in this pull request?

Create add-on for `ozone` docker-compose environment to demonstrate integration with [Iceberg](https://iceberg.apache.org/) and [Trino](https://trino.io/).

https://issues.apache.org/jira/browse/HDDS-8971

## How was this patch tested?

Added test script to verify the setup:
 * create S3 bucket in Ozone
 * create table and insert data in `spark-shell` (example taken from [Spark and Iceberg Quickstart](https://iceberg.apache.org/spark-quickstart/))
 * describe table and insert more data in `trino`
 * check that data/metadata is stored in Ozone

CI:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/5437385105
(Interesting part begins [here](https://github.com/adoroszlai/hadoop-ozone/actions/runs/5437385105/jobs/9888115026#step:5:102).)
